crud/select/comparators.lua

```diff
@@ -158,23 +167,29 @@ local array_cmp_funcs_by_operators = {
     [operators.GE] = array_ge,
 }
 
-function comparators.gen_func(operator, key_parts)
-    local cmp_func = array_cmp_funcs_by_operators[operator]
+function comparators.get_cmp_operator(tarantool_iter)
+    local cmp_operator = cmp_operators_by_tarantool_iter[tarantool_iter]
+    assert(cmp_operator ~= nil, 'Unknown Tarantool iterator %q', tarantool_iter)
+
+    return cmp_operator
+end
 
+function comparators.gen_func(cmp_operator, key_parts)
+    local cmp_func = array_cmp_funcs_by_operators[cmp_operator]
     if cmp_func == nil then
-        return nil, ComparatorsError:new('Unsupported operator %q', operator)
+        return nil, ComparatorsError:new('Unsupported operator %q', cmp_operator)
```

Comment:
Still I'm sure "Unsupported" and "Unknown" iterators should be splitted

Reply:
It's not a check that iterator isn't exists, it's an only check that it can be used for tuples comparator, so it's `Unsupported`.